build.rs

```diff
@@ -6,5 +6,8 @@ fn main() {
     // Set features depending on channel
     if version().unwrap() >= Version::parse("1.37.0").unwrap() {
         println!("cargo:rustc-cfg=reverse_bits");
+        if version().unwrap() >= Version::parse("1.85.0").unwrap() {
```

Comment:
I *think* but I might be wrong this can be:

```rust
if version > 1.85.0 { println!("...") }
else if version > 1.37.0 { println!("...") }
```

but I might be wrong!

Reply:
It at least works on 1.85 if you don't print the other.

Changed in [a8b2625](https://github.com/softdevteam/vob/pull/73/commits/a8b262562eb5ec1c2c91b07ca757eff6598c5f2d)

Reply:
Oh wait hold up, I believe this other line is *changing* the cfg value,
e.g. it is equivalent to RUSTFLAGS="--cfg reverse_bits".

while `rustc-check-cfg` is just silencing a warning that you are using a `#[cfg(foo)]` that rustc doesn't know about (whether `cfg(reverse_bits)` is set or not.

So I think this changes the default of whether that `--cfg` is passed to depend on the compiler.

I don't think that is what we want?

Reply:
My memory on this is fuzzy, but I think `reverse_bits` was stabilised at some point? Maybe it doesn't even need mentioning at all for recent Rust versions?

Reply:
I *think* what is going on is that `reverse_bits` is a *vob* config option that turns on the fast path.
If we remove the emitting of that line we revert back to the slow path on 1.85 and later.

So I think we want both `println` on 1.85 and just the original `1.37` and later

Reply:
Or maybe consider removing the slow path code, if it has been long enough?

Reply:
I think there are two aspects:

1. `reverse_bits` turns on a fast path by using the `reverse_bits` function in std which was stabilised in 1.37.
2. The way we tell Cargo to turn on that fast path has changed.

I tend to agree that supporting < 1.37 (Aug 2019) no longer feels worthwhile. I will raise a PR to remove the slow path.

Reply:
Great, I'll just remove the `build.rs` changes from this patch and squash, leaving it to that PR.